torch/csrc/distributed/rpc/rref_impl.h

```diff
@@ -213,6 +213,8 @@ class TORCH_API RRef : public RRefInterface {
     return type_;
   }
 
+  virtual void tryDel() {}
```

Comment:
Let's add comments to explain its behavior.

Reply:
Adding.